<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_offer_dialog</class>
 <widget class="QDialog" name="view_offer_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View Offer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="viewOfferGroupBox">
       <property name="title">
        <string>View Offer: Offername</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPlainTextEdit" name="descPlainTextEdit">
            <property name="lineWrapMode">
             <enum>QPlainTextEdit::NoWrap</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="vendorNameLabel">
            <property name="text">
             <string>Vendor: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ratioLabel">
            <property name="text">
             <string>Refund Ratio: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="locktimeLabel">
            <property name="text">
             <string>Locktime: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="minRepLabel">
            <property name="text">
             <string>Min. Reputation Score: </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="descLabel">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="tagsGroupBox">
       <property name="title">
        <string>Offer tags:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tagsTableWidget">
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Description</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
